devdoc/SecurityConfig.md

```diff
@@ -0,0 +1,83 @@
+# Security Configuration #
+This document covers security configuration values and meanings. It does not server as
+the source for how to configure security, but rahter what the different values mean. These
+are not covered in the docs nor does any reasonable help information exist in the
+config manager or yaml files.
+
+## Supported Configurations ##
+Security is configured as either through an LDAP server or a plugin. Additionally, these are
+supported in both legacy deployments that use _environment.xml_ and containerized deployments 
+using Kubernetes and helm charts. While these methods differ, the configuration values
+remain the same. Focus is placed on the different values and not the deployment
+target. Differences based on deployment can be found in the relevant platform documents.
+
+## Security Managers ##
+Security is implemented via a security manager interface. Managers are loaded and used by components within the
+system to check authorization and authentication. LDAP is an exception to the loadable manager model. It is not a compliant
+loadable module like other security plugins. For that reason, the configuration for each is separated into two 
+sections below: LDAP and Plugin Security Managers.
+
+### LDAP ###
+LDAP is a protocol that connects to an Active Driectory server (AD). The term LDAP is used
+interchangeably with AD. Below are the configuration values for an LDAP connection. These are valid for both legacy
```

Comment:
Just a side comment, but "interchangeably with AD." I guess is true but that's a bit unfortunate.  I wish we could use other LDAP backends especially if we could have a stand alone one that easily worked on a single developer laptop helm deployment.